Mellanox/dpdk.org
# For next

- Add latest commits from dpdk/master
 - Remove Oleg's timestamp commits
 - Update tx doorbell patch

checkpatch is not expected to pass due to already accepted patches (same issue for check-git-log.sh)
